### DISCUSSION TITLE
v2.0.35

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#371] [Announcements]
## Changes

- Make absorption a configurable setting @DTTerastar (#302)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.0.35'>v2.0.35</a>.</em>

**Comment by Zipties** [comment]
Could you explain a little on how and why absorption would be configured?

**Reply by V1d1o7** [reply]
I was thinking the same thing, but personally thought "why ask here I am sure it's documented on the site or in the mentioned issue... surprised to find no documentation on it. I am very interested in what absorption is and why one may want it a configurable setting.

**Reply by DTTerastar** [reply]
This is a good write up: https://github.com/ESPresense/ESPresense/issues/324
